module/main.py

```diff
@@ -0,0 +1,58 @@
+import turtle, line, rectangle, polygon, circle, ellipse
+
+
+class Figures:
+    def __init__(self, x, t):
+        self.__x = x
+        if x == 1:
+            self.Line = line.Line(int(input('Enter line size:\n')),t)
+
+        elif x == 2:
+            x = int(input('Enter width size:\n'))
+            y = int(input('Enter height size:\n'))
+            self.Rectangle = rectangle.Rectangle([x, y, x, y], t)
+
+        elif x == 3:
+            self.Polygon = polygon.Polygon(int(input('Enter side size:\n')), int(input('Enter side amount:\n')), t)
+
+        elif x == 4:
+            self.Circle = circle.Circle(int(input('Enter radius:\n')), t)
+
+        elif x == 5:
+            self.Ellipse = ellipse.Ellipse(int(input('Enter radius:\n')), t)
+
+    def draw(self):
+        if self.__x == 1:
+            self.Line.draw()
+
+        if self.__x == 2:
+            self.Rectangle.draw()
+
+        if self.__x == 3:
+            self.Polygon.draw()
+
+        if self.__x == 4:
+            self.Circle.draw()
+
+        if self.__x == 5:
+            self.Ellipse.draw()
```

Comment:
if self.__x == 6:
if self.__x == 7:
if self.__x == 8:
if self.__x == 9:
....
if self.__x == n:
if self.__x == n+1:
Your opinion. Is it ok?